requirements.txt

```diff
@@ -1,6 +1,6 @@
 requests==2.26.0
 pytest==6.2.5
 maturin
-uvloop
+uvloop; platform_system!="Windows"
```

Comment:
niceee.

Reply:
I didn't know that was a thing until I worked on this PR :smile: